ui/component/swipeableDrawer/view.jsx

```diff
@@ -31,18 +32,29 @@ export default function SwipeableDrawer(props: Props) {
 
   const [coverHeight, setCoverHeight] = React.useState();
 
-  const videoHeight = coverHeight || (mobilePlayerDimensions ? mobilePlayerDimensions.height : 0);
+  const videoHeight = (mobilePlayerDimensions && mobilePlayerDimensions.height) || coverHeight || 0;
+
+  const handleResize = React.useCallback(() => {
+    const element =
+      document.querySelector(`.${PRIMARY_IMAGE_WRAPPER_CLASS}`) ||
+      document.querySelector(`.${PRIMARY_PLAYER_WRAPPER_CLASS}`);
+
+    if (!element) return;
+
+    const rect = element.getBoundingClientRect();
+    setCoverHeight(rect.height);
+  }, []);
 
   React.useEffect(() => {
-    if (open && !mobilePlayerDimensions) {
-      const element = document.querySelector(`.file-page__video-container`);
+    // Drawer will follow the cover image on resize, so it's always visible
+    if (open && (!mobilePlayerDimensions || !mobilePlayerDimensions.height)) {
+      handleResize();
+
+      window.addEventListener('resize', handleResize);
```

Comment:
Enhancement:  Might need to debounce these .. I know most of existing ones are doing it either.
Asad's `useOnResize` may or may not be useful (I think it doesn't handle "conditionally register the handle" ... so maybe just debounce the handler?)